### ISSUE TITLE
Did you just totally take 7 epochs to get the final training accuracy 60%+ in the CNN_mini_project(py3.5).ipynb?I'm new so I may not get it right...

### ISSUE THREAD

**Issue by AAApo** [Hanzhuo/Facial-Expression-Recognition-with-TensorFlow-Convolutional-Neural-Networks#1]

**Comment by Hanzhuo** [comment]
I am sorry to reply so late. The original Tensoflow code (https://github.com/Hanzhuo/Facial-Expression-Recognition-with-TensorFlow-Convolutional-Neural-Networks/blob/master/CNN_Facial_Expression_Recogonition.ipynb) can reach 60%+. If you use keras code you have to change 7 epochs to much more epochs.

**Comment by AAApo** [comment]
Thanks a lot




------------------ 原始邮件 ------------------
发件人: "Hanzhuo"<notifications@github.com>; 
发送时间: 2017年5月26日(星期五) 中午11:19
收件人: "Hanzhuo/Facial-Expression-Recognition-with-TensorFlow-Convolutional-Neural-Networks"<Facial-Expression-Recognition-with-TensorFlow-Convolutional-Neural-Networks@noreply.github.com>; 
抄送: "曾昱雯—B4"<891775008@qq.com>; "Author"<author@noreply.github.com>; 
主题: Re:[Hanzhuo/Facial-Expression-Recognition-with-TensorFlow-Convolutional-Neural-Networks]Did you just totally take 7 epochs to get the final training accuracy 60%+ inthe CNN_mini_project(py3.5).ipynb?I'm new so I may not get it right... (#1)




I am sorry to reply so late. The original Tensoflow code (https://github.com/Hanzhuo/Facial-Expression-Recognition-with-TensorFlow-Convolutional-Neural-Networks/blob/master/CNN_Facial_Expression_Recogonition.ipynb) can reach 60%+. If you use keras code you have to change 7 epochs to much more epochs.
